c7n/resources/sagemaker.py

```diff
@@ -140,6 +140,26 @@ def _augment(j):
         return list(map(_augment, super(SagemakerTransformJob, self).augment(jobs)))
 
 
+@resources.register('sagemaker-labeling-job')
+class SagemakerLabelingJob(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        arn_type = "labeling-job"
+        service = 'sagemaker'
+        enum_spec = ('list_labeling_jobs', 'LabelingJobSummaryList', None)
+        detail_spec = (
+            'describe_labeling_job', 'LabelingJobName', 'LabelingJobName', None)
+        arn = id = 'LabelingJobArn'
+        name = 'LabelingJobName'
+        date = 'CreationTime'
+        universal_taggable = object()
+        permission_augment = ('sagemaker:DescribeLabelingJob', 'sagemaker:ListTags')
+
+    def augment(self, resources):
```

Comment:
Does this even pull the tags? I was working on few other sagemaker resources and found that we would have to do list_tags() to actually pull them.

Reply:
Hey Pratyush, good question. The describe response has it for this one https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.describe_labeling_job.

Reply:
I had to do this in augment to get the describe for each one though. 
```python
        return universal_augment(
            self, super().augment(resources))
```

Reply:
ah, nvm, describe call in this case is returning the tags too which was not the case with some other sagemaker resources. Should be good here!!